src/main/scala/io/iohk/ethereum/db/storage/BlockHeadersNumbersStorage.scala

```diff
@@ -0,0 +1,17 @@
+package io.iohk.ethereum.db.storage
+
+import akka.util.ByteString
+import io.iohk.ethereum.db.dataSource.DataSource
+
+class BlockHeadersNumbersStorage(val dataSource: DataSource) extends KeyValueStorage[BigInt, ByteString] {
```

Comment:
Is the storage `BlockNumberMappingStorage` used for the same purpose? Maybe we should keep only one of them.